tests/codebuild/run_integtest.sh

```diff
@@ -78,40 +103,101 @@ else
     readonly cluster_region="$(echo "${cluster_info}" | awk '{print $2}')"
 fi
 
-
+echo "SETUP FOR INTEGRATION TESTS"
 # Download the CRD from the tarball artifact bucket
 aws s3 cp s3://$ALPHA_TARBALL_BUCKET/${CODEBUILD_RESOLVED_SOURCE_VERSION}/sagemaker-k8s-operator-us-west-2-alpha.tar.gz sagemaker-k8s-operator.tar.gz 
 tar -xf sagemaker-k8s-operator.tar.gz
-
 # Jump to the root dir of the operator
 pushd sagemaker-k8s-operator
-
-    # Setup the PATH for smlogs
+# Setup the PATH for smlogs
     mv smlogs-plugin/linux.amd64/kubectl-smlogs /usr/bin/kubectl-smlogs
-
-    # Allow for overriding the installation of the CRDs/controller image from the
-    # build scripts if we want to use our own installation
-    if [ "${SKIP_INSTALLATION}" == "true" ]; then
-        echo "Skipping installation of CRDs and operator"
-    else
-        # Goto directory that holds the CRD  
-        pushd sagemaker-k8s-operator-install-scripts
-            # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
-            ./setup_awscreds
-
-            echo "Deploying the operator"
-            kustomize build config/default | kubectl apply -f -
-        popd
-
-        echo "Waiting for controller pod to be Ready"
-        # Wait to increase chance that pod is ready
-        # TODO: Should upgrade kubectl to version that supports `kubectl wait pods --all`
-        sleep 60
-    fi 
+    pushd sagemaker-k8s-operator-install-scripts
+        # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
+        ./setup_awscreds
+        path_to_installer=$(pwd)
+    popd
 popd
 
-# Run the integration test file
-cd tests/codebuild/ && ./run_all_sample_test.sh
+echo "RUN INTEGRATION TESTS FOR THE CLUSTER SCOPED OPERATOR DEPLOYMENT"
```

Comment:
Debugging output?

Reply:
makes it easier to parse the logs but can remove or switch to lowercase if it's an eyesore.

Reply:
Just for consistency, I think it would be good to make it sentence cased. Could even add a square bracket header before it like `"[SECTION] Running integration tests for cluster scoped operator deployment"`.

Reply:
Sounds good. Done.